docs/docs/rest-api/public/api/v2/events.raml

```diff
@@ -0,0 +1,33 @@
+get:
+  description:
+    Attach to the marathon event stream.
```

Comment:
This is a question of your expectation: 
- if you add the Accept Header, the Try it out function connects to the event stream and will not return -> you see nothing
- if you omit the Accept Header, you get a 405 response.

That means this functionality can not be used in the console. 
I will add a sentence to the docs.